README.md

```diff
@@ -9,73 +9,70 @@ the square brackets (for multiple-choice questions).
 4. Add/Commit/Push your changes to Github.
 5. Open a pull request.
 
-## CSS
+## HTML
 
 ### Question #1
 
-Describe the purpose of a clearfix in CSS, and give an example of how to do it.
+What does it mean for an HTML tag to be semantic? Give an example of one.
```

Comment:
How about, give an example of a short snippet of HTML written using semantic tags and non-semantic tags?

(Maybe this variant is too long, but I think it's important to go beyond 'this is a semantic' tag, to an actual example.)